[4.0] Update Readme for Api tests (#30539)
joomla/joomla-cms: tests/Codeception/README.md

### Installation

> 1) Run a `composer install` in the Joomla main directory 

Reviewer: ```suggestion
1) Run a `composer install` in the Joomla main directory.
```

> 2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then adjust the REST url in the new file.

Reviewer 2: ```suggestion
2) Copy the file `tests/Codeception/api.suite.dist.yml` to `tests/Codeception/api.suite.yml`. Then edit the REST url in the new file to point it to your localhost url.
```

> 3) Make sure, that the file `codeception.yml` is in the Joomla main directory. 

Reviewer: Use `main` or `root` and capitalize Joomla as in 1 for consistency?

Author: Thank you Quy

> 3) Make sure, that the file `codeception.yml` is in the Joomla main directory. 

Reviewer: ```suggestion
3) Be sure that the file `codeception.yml` is in the Joomla main directory. 
```

Author: Thanks
